src/promptflow/tests/sdk_cli_test/unittests/test_local_storage_operations.py

```diff
@@ -17,12 +17,20 @@ def test_outputs_padding(self) -> None:
             {LINE_NUMBER: 2, "col": "b"},
         ]
```

Comment:
I remember there are cases that data is not fully aligned if some col contains error/not return any values.

Will you create case for these situations?

Reply:
the newly added case (second part of this test) targets that